spec/controllers/reservations_controller_spec.rb

```diff
@@ -1656,6 +1604,10 @@
   end
 
   describe '#deny_request PUT' do
+    # SMELL: this doesn't belong in the controller
```

Comment:
Since this is repeated above in line 1583, could we create something shared between both?